step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesExecutionPlugin.java

```diff
@@ -1,16 +1,30 @@
 package step.plugins.timeseries;
 
+import step.core.artefacts.reports.ReportNodeStatus;
 import step.core.execution.ExecutionContext;
 import step.core.execution.ExecutionEngineContext;
 import step.core.execution.model.Execution;
+import step.core.execution.model.ExecutionAccessor;
 import step.core.plugins.Plugin;
+import step.core.timeseries.TimeSeriesIngestionPipeline;
+import step.core.timeseries.bucket.BucketAttributes;
+import step.core.views.ViewManager;
 import step.engine.plugins.AbstractExecutionEnginePlugin;
+import step.plugins.views.functions.ErrorDistribution;
+import step.plugins.views.functions.ErrorDistributionView;
+
+import java.util.HashMap;
+import java.util.Map;
 
 @Plugin(dependencies= {})
```

Comment:
@david-stephan The TimeSeriesIngestionPipeline.class is added to the context by the TimeSeriesControllerPlugin at server start. So we're sure it will be present in the context. Only issue that could happen is that the TimeSeriesControllerPlugin isn't enabled. As both plugins are in the same module this is not likely to happen. Unfortunately we cannot add a dependency to a controller plugin from an execution plugin